### PR TITLE
[CAMEL-17878] - add tests in camel-kamelet-starter.

### PR DESCRIPTION
**Pull request by VaishnaviR18**
https://issues.apache.org/jira/browse/CAMEL-17878 
Covers tests for camel-kamelet-starter (ported the tests from camel-kamelet).

Thanks!

### CODE REVIEW

File: components-starter/camel-kamelet-starter/pom.xml [hunk 1]
```diff
@@ -38,6 +38,78 @@
       <artifactId>camel-kamelet</artifactId>
       <version>${camel-version}</version>
     </dependency>
+    <!-- Testing dependencies -->
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-xml-jaxb</artifactId>
+      <scope>test</scope>
```
Comment (davsclaus) [inline components-starter/camel-kamelet-starter/pom.xml:45]: Add

    <version>${camel-version}</version>

for all dependency you add that has groupId = org.apache.camel

**Review by davsclaus — CHANGES_REQUESTED** [review]

File: components-starter/camel-kamelet-starter/pom.xml (hunk 1 above)
Comment (VaishnaviR18) [reply components-starter/camel-kamelet-starter/pom.xml:45]: Okay, thanks!

Comment (VaishnaviR18) [reply components-starter/camel-kamelet-starter/pom.xml:46]: @davsclaus I have added it. Can you please review it now?
Thanks!

**Review by davsclaus — APPROVED** [review]

**Comment by VaishnaviR18** [comment]
Thanks!

**Pull request merged**